docs/reference/ACL-migration.md

```diff
@@ -0,0 +1,69 @@
+### The new ACL model
+As mentioned in our [Security Update](https://docs.lakefs.io/posts/security_update.html#whats-changing) the coming lakeFS version introduces a simplified authorization mechanism based on ACLs.
+
+### Permissions
+In the ACL model Every group will have a permission attached to it,
+lakeFS provides the following permissions:
+
+| Permission | Allows | Existing Group |
+|------------|--------------------------------------------|---------------------------|
+| **Read** | Read operations, creating access keys. | Viewers |
+| **Write** | Allows all data read and write operations. | Developers |
+| **Super** | Allows all operations except auth. | SuperUsers (with changes) |
+| **Admin** | Allows all operations. | Admins |
+
+### Scopes
+
+When granted to a group, permissions Read, Write, and Super may be scoped to a set of repositories.
+Admin includes global abilities that apply across repos and cannot be scoped to a set of repositories
+
+### ACL
+
+You can now attach Permissions and scope them to groups in the Groups page
+There are 4 default groups, named after the 4 permissions. Each group is global (applies for all repositories).
+The group list initially would be
+
+| Group ID    | Default Permission | Created at       | Repositories |
+|-------------|--------------------|------------------|--------------|
+| **Read**    | Read               | \<Creation date> | All          |
+| **Write**   | Write              | \<Creation date> | All          |
+| **Super**   | Super              | \<Creation date> | All          |
+| **Admin**   | Admin              | \<Creation date> | All          |
+
+
+In the UI:
+
+![Groups page has a dropdown to edit permission for each
+group]( ../assets/img/ACL-groups-with-perms.png)
+
+
+
+### The migration
+
+Upgrading the lakeFS version will require running a migrating to our ACL authorization model in case:
```

Comment:
```suggestion
Upgrading the lakeFS version will require migrating to the new ACL authorization model in case:
```

Reply:
Isn't it required anyway? and requires manual intervention if the following actions took place?

Reply:
Yup, always required.  (Technically it will might with no update, but you will be unable to edit any policies or even see what they are.)